### ISSUE TITLE
Update for Swift 2?

### ISSUE THREAD

**Issue by nickbewley** [rakaramos/StarButton#2]
Hey there,

Novice in iOS development so apologies if this question seems dense. Thanks for writing the tutorial, but I have encountered some errors after auto-converting (via Xcode) the project to Swift 2.0 and am at a loss for how to fix them. Can anyone help guide me here?

These errors I auto-corrected using the recommended Xcode patch:

```
Errors #2-5: "Value of optional type .. not unwrapped; did you mean to use '!' or '?'" (Extensions.swift lines 44, 49, 73
Error #6: "Initialization of immutable value 'starAnimation' was never used; consider replacing with assignment to '_'"
```

This is the error that remains and causes the build to fail:

```
Error #1: "Use of unresolved identifier 'countElements'" (Extensions.swift line 19)
```

Thanks for your help here!

**Comment by rakaramos** [comment]
@nickbewley thanks for pointing that out! I'm going to fix this issue son :)

**Comment by nickbewley** [comment]
Sounds great. Thanks @rakaramos!

**Comment by rakaramos** [comment]
fixed on commit 2b4445b

**Comment by nickbewley** [comment]
Thanks a ton for the quick patch @rakaramos! This tutorial is a huge help for newbs like me!
